src/Buerokratt.Common.UnitTests/ServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Buerokratt.Common.CentOps;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using Xunit;
+
+namespace Buerokratt.Common.UnitTests
```

Comment:
Could you put tests related to CentOps object under a "CentOps" directory please? It would be easier to navigate and we wouldn't have to start renaming files when other ServiceCollection extensions are introduced for other purposes/classes

Reply:
Yep - good idea.